src/librustc_ast/ast.rs

```diff
@@ -314,35 +314,32 @@ pub type GenericBounds = Vec<GenericBound>;
 pub enum ParamKindOrd {
     Lifetime,
     Type,
-    // unordered is only true if sess.has_features().const_generics
-    // is active. Specifically, if it's only `min_const_generics`, it will still require ordered
-    // types.
+    // `unordered` is only `true` if `sess.has_features().const_generics`
+    // is active. Specifically, if it's only `min_const_generics`, it will still require
+    // ordering `const`s after `type`s.
```

Comment:
```suggestion
    // ordering consts after types.
```
const and type aren't variables here :laughing:

Reply:
ah, I just assumed that anything that was a technical term was \` \`